### PR TITLE
[sailfish-office] Calculate number of visible cover list items dynami…

### PR DESCRIPTION
**Pull request by jpetrell**
…cally. Contributes to JB#14363

### CODE REVIEW

File: qml/CoverPage.qml [hunk 1]
```diff
@@ -45,22 +45,29 @@ CoverBackground {
...
+            bottomMargin: Theme.paddingLarge
+        }
 
         delegate: CoverFileItem {
             width: fileListView.width
             height: fileListView.itemHeight
             text: model.fileName
             iconSource: window.mimeToIcon(model.fileMimeType)
         }
+        Label {
```
Comment (dcaliste) [inline qml/CoverPage.qml]: I don't know if it's a good suggestion, but anyway:

Would it be better to create here a dummy CoverFileItem instance, so like that if later on, someone who makes changes in the layout of CoverFileItem (and its height is not a label height anymore), won't have to update here also ?

This suggestion, though requires to add ```height: label.height``` in CoverFileItem.qml.

**Comment by pvuorela** [comment]
LGTM

File: qml/CoverPage.qml (hunk 1 above)
Comment (jpetrell) [inline qml/CoverPage.qml]: Currently height is explicitly determined by the user of CoverFileItem so I think deriving label height from external item is also ok. That being said I wouldn't normally write layouts like this (implicit height is normally defined by the item itself), but rewriting the layout feels a bit of an overkill too. Adding invisible dummy CoverFileItem would add more loading overhead than adding just a label, I will replace the label with FontMetrics QML element once Qt 5.6 is in use.

Comment (dcaliste) [inline qml/CoverPage.qml]: I thank you for the explanation.

**Pull request merged**